Makefile

```diff
@@ -1,3 +1,10 @@
+all: discovery node
 discovery:
 	go build ./cmd/discovery
 	@echo "Done discovery building"
+
+node:
+	go build -o seele-node ./cmd/node 
```

Comment:
please don't specific output file. For different os, the file extension is different

Reply:
oh，I forgot other OS like windows. I modify the output file name because the default is conflicted with the directory "node" in the project root dir. May be we can check OS or add a bin directory and put the compiled files in this dir?

Reply:
Anyway, I think we could only support linux env